util/string.go

```diff
@@ -227,7 +227,7 @@ func IsChinese(str string) bool {
 }
 
 func GetMaskedPhone(phone string) string {
-	return getMaskedPhone(phone)
+	return rePhone.ReplaceAllString(phone, "$1****$2")
```

Comment:
Why not remove getMaskedPhone?

Reply:
`getMaskedPhone` no usage found in other places